### PR TITLE
docs(transparentproxy): add info about adjusting exclude inbound/outbound ports

### PR DESCRIPTION
**Pull request by slonka**
this came as a question on slack

Did you sign your commit? [Instructions](https://github.com/kumahq/.github/blob/main/CONTRIBUTING.md#sign-your-commits) yes

Have you read [Contributing guidelines](https://github.com/kumahq/.github/blob/main/CONTRIBUTING.md)? yes

### CODE REVIEW

File: app/_src/production/dp-config/transparent-proxying.md [hunk 1]
```diff
@@ -60,7 +60,7 @@ The host that will run the `kuma-dp` process in transparent proxying mode needs
    useradd -u 5678 -U kuma-dp
    ```
 
-3. Redirect all the relevant inbound, outbound and DNS traffic to the {{site.mesh_product_name}} data plane proxy.
+3. Redirect all the relevant inbound, outbound and DNS traffic to the {{site.mesh_product_name}} data plane proxy (if you're running any other services on that machine you need to adjust the comma separated lists of `--exclude-inbound-ports` and `--exclude-outbound-ports` accordingly).
```
Comment (lahabana) [inline app/_src/production/dp-config/transparent-proxying.md:63]: Isn't the right config something with uid too?

Comment (slonka) [reply app/_src/production/dp-config/transparent-proxying.md:63]: would you like to exclude it from redirects completely? like with `--kuma-dp-uid`? we'd need another switch for that like `--redirect-disabled-uids`?

Comment (slonka) [reply app/_src/production/dp-config/transparent-proxying.md:65]: @lahabana WDYT?

**Review by Automaat — APPROVED** [review]

File: app/_src/production/dp-config/transparent-proxying.md (hunk 1 above)
Comment (slonka) [reply app/_src/production/dp-config/transparent-proxying.md:65]: let's discuss it in an issue https://github.com/kumahq/kuma/issues/10672

**Pull request merged**